apps/snitch_core/priv/repo/migrations/20181024091825_add_postgres_money_aggregate_functions.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Snitch.Repo.Migrations.AddPostgresMoneyAggregateFunctions do
```

Comment:
was this added when you ran the task to migrate for `Money`

Reply:
yes it is added after I ran the migration.